src/utils/dates.js

```diff
@@ -40,3 +40,39 @@ export function getDaysBetweenDates(startDate, endDate) {
 		console.error('An error happened', error);
 	}
 }
+
+export function purchaseSchedule(dateLastPurchased, dateNextPurchased) {
+	const today = new Date();
+	const dateNextPurchased_formatted = dateNextPurchased.toDate();
```

Comment:
nit: try to be consistent with your casing. We're mixing camelCase and snake_case here

Reply:
Thank you for the suggestion Devin, we've changed it naming to camelCase `dateNextPurchasedFormatted`